utils/collectd/files/lua-scripts/dsl.lua

```diff
@@ -105,11 +121,17 @@ end
 local function get_values(hostname, variables, metrics, direction)
 	for _, information in pairs(variables) do
 		local name = information["name"]
+		local type = information["type"]
 
 		if metrics and metrics[name] ~= nil then
 			local value = metrics[name]
-			local metric = build_metric(name, direction)
-			if information["type"] == "bool" then
+
+			if name = type then
```

Comment:
Yup, you're right. That's just a typo. As I said, I haven't yet tested these changes lol.

You're probably right. I mean, it doesn't really make any difference so it could also just be a waste of time to put any more thought into it. Ultimately, it ain't my call. If you want this change removed, I'll happily do so.